src/vs/workbench/api/browser/mainThreadChatSessions.ts

```diff
@@ -409,6 +411,13 @@ export class MainThreadChatSessions extends Disposable implements MainThreadChat
 			originalGroup = this.editorGroupService.activeGroup;
 		}
 
+		const options: IChatEditorOptions = {
+			title: {
+				preferred: originalEditor?.getName() || undefined,
```

Comment:
would it be better if we just fired off label change on the initial `resolve` call in the ChatEditorInput for this scenario?